pr_agent/tools/pr_similar_issue.py

```diff
@@ -194,8 +190,8 @@ def _update_index_with_issues(self, issues_list, repo_name_for_index, upsert=Fal
 
             issue_str, comments, number = self._process_issue(issue)
             issue_key = f"issue_{number}"
```

Comment:
**Suggestion:** Consider using a helper function to process the issue, as this operation is repeated multiple times in the code. This will make the code cleaner and easier to maintain. [maintainability]